documentation/deployments.md

```diff
@@ -11,3 +11,51 @@ Upon merging a PR to `main` a deployment to the development environment will be
 1. Initialise Terraform
 1. Run the Terraform plan
 1. Apply the Terraform changes
+
+### PaaS
+
+Some useful things to know about PaaS...
+
+GOV.UK Platform as a Service (PaaS) is a cloud-hosting platform built by the Government Digital Service (GDS). GOV.UK PaaS manages the deployment of your apps, services and background tasks so you don’t need to hire people with specialist cloud skills.
+
+GOV.UK PaaS uses the open source Cloud Foundry project, and runs on Amazon Web Services.
+
+This repo has an organisation setup on PaaS. Speak to the org manager to authorise the [creation of a personal account](https://docs.cloud.service.gov.uk/get_started.html#get-started)
+
+The Cloud Coundry CLI can be used to manage apps hosted on PaaS - [Setup Cloud Foundry CLI](https://docs.cloud.service.gov.uk/get_started.html#set-up-the-cloud-foundry-command-line)
+
+### Environments
+
+This application runs through four environments:
+
+- dev
+- test
+- pre prod
+- prod
+
+`dev` is deployed to automatically when a PR is merged into the `main` branch.
+
```

Comment:
If the database is destroyed when a main branch is deployed, can we warn about it here ?   

Changes made by  an editor will be lost.

Reply:
2fc0f3f